### PR TITLE
Simplify category sources config

### PR DESCRIPTION
**Pull request by rustamwin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️/❌
| Breaks BC?    | ✔️/❌
| Fixed issues  | Support upcoming release of `yiisoft/definitions`

### CODE REVIEW

**Review by vjik — CHANGES_REQUESTED** [review]
Need test

File: config/common.php [hunk 1]
```diff
@@ -11,13 +11,18 @@
...
     // Configure application CategorySource
-    // 'translator.app' => static function (\Yiisoft\Translator\IntlMessageFormatter $formatter) use ($params) {
-    //     return new \Yiisoft\Translator\CategorySource(
-    //         $params['yiisoft/translator']['defaultCategory'],
-    //         new \Yiisoft\Translator\Message\Php\MessageSource('/path/to/messages'),
-    //         $formatter
-    //     );
-    // },
+    //'translation.app' => [
+    //    'definition' => static function (Aliases $aliases, MessageFormatterInterface $messageFormatter) use ($params) {
```
Comment (vjik) [inline config/common.php:15]: 1) `MessageFormatterInterface` don't should defined in container in base cases. 
2) `Aliases` can be don't used in application. Suggest don't use additional dependencies in example.

File: config/common.php [hunk 2]
```diff
@@ -11,13 +11,18 @@
...
+    //        $messageSource = new MessageSource($aliases->get('@messages'));
+    //
+    //        return new CategorySource(
```
Comment (vjik) [inline config/common.php:18]: Suggest use FQN for ability copy/paste example.

**Comment by rustamwin** [comment]
> Need test

Added

**Review by vjik — APPROVED** [review]

**Review by samdark — APPROVED** [review]

**Pull request merged**